test/ParseableInterface/ModuleCache/prebuilt-module-cache-unusable.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/MCP)
+// RUN: %empty-directory(%t/prebuilt-cache/Lib.swiftmodule)
+
+// Set OTHER_CPU to x86_64, unless we're using x86_64, at which point use arm64.
+// It doesn't matter what these are, as long as OTHER_CPU is any supported
+// architecture other than the current one we're testing.
+
+// RUN: OTHER_CPU=""
+// RUN: if [ '%target-cpu' = 'x86_64' ]; then OTHER_CPU='arm64'; else OTHER_CPU='x86_64'; fi
```

Comment:
This won't work on Windows, where we don't have a shell. Is there a problem using something imaginary?